lib/src/util.dart

```diff
@@ -1,4 +1,72 @@
 import 'dart:convert';
 
+import 'package:charcode/charcode.dart';
+
 String escapeHtml(String html) =>
     const HtmlEscape(HtmlEscapeMode.ELEMENT).convert(html);
+
+// Escape the contents of [value], so that it may be used as an HTML attribute.
+
+// Based on http://spec.commonmark.org/0.28/#backslash-escapes.
+String escapeAttribute(String value) {
+  var result = new StringBuffer();
+  var codeUnits = value.codeUnits;
+  int ch;
+  for (var i = 0; i < codeUnits.length; i++) {
+    ch = codeUnits[i];
+    if (ch == $backslash) {
```

Comment:
I don't understand why there's backslash-specific logic here. Backslashes don't mean anything in particular to HTML, do they?

Reply:
This is here to support the "that's not an escape sequence" case: http://spec.commonmark.org/0.28/#example-473

But I think you're right. Makes more sense to do this logic while parsing a link, rather than here.